src/scripts/main.js

```diff
@@ -358,3 +358,33 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.getElementsByClassName('dashboard')[0];
+const innerBody = table.lastElementChild;
+
+for (const person of people) {
+  innerBody.appendChild(document.createElement('tr'));
+
+  const line = innerBody.lastElementChild;
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = person.name;
+  line.appendChild(document.createElement('th'));
+
+  if (person.sex === 'm') {
+    line.lastElementChild.textContent = 'Male';
+  } else if (person.sex === 'f') {
+    line.lastElementChild.textContent = 'Female';
+  }
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = person.born;
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = person.died;
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = person.died - person.born;
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = Math.ceil(person.died / 100);
+}
```

Comment:
You are directly manipulating the DOM within a loop, which is inefficient and can lead to performance issues, especially when dealing with large data sets. It's better to create a document fragment, append your elements to that, and then append the document fragment to the DOM. This way, the browser only has to reflow and repaint once, rather than each time an element is added.